### PR TITLE
chunked: improve error messages for file system operations

### PR DESCRIPTION
**Pull request by giuseppe**
follow-up for https://github.com/containers/storage/pull/1952

@mtrmac PTAL

### CODE REVIEW

**Review by mtrmac — COMMENTED** [review]
LGTM, just one stylistic nit.

File: pkg/chunked/filesystem_linux.go [hunk 1]
```diff
@@ -35,12 +36,27 @@ type fileMetadata struct {
...
+	destDir, destBase := filepath.Split(destFile)
+	destDirFd := dirfd
+	if destDir != "" && destDir != "." {
+		f, err := openOrCreateDirUnderRoot(dirfd, destDir, 0)
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+		destDirFd = int(f.Fd())
+	}
```
[the remark this answers is not in the document]
Comment (giuseppe) [reply pkg/chunked/filesystem_linux.go:62]: that is a good idea for a further refactoring.

**Comment by giuseppe** [comment]
rebased

**Pull request merged**